Sources/protoc-gen-swift/ReservedWords.swift

```diff
@@ -211,60 +127,23 @@ func sanitizeFieldName(_ s: String) -> String {
  * Many Swift reserved words can be used as enum cases if we put
  * backticks around them:
  */
-private let quotableEnumCases: Set<String> = [
-            "as",
-            "associativity",
-            "break",
-            "case",
-            "catch",
-            "class",
-            "continue",
-            "default",
-            "defer",
-            "deinit",
-            "do",
-            "dynamicType",
-            "else",
-            "enum",
-            "extension",
-            "fallthrough",
-            "false",
-            "for",
-            "func",
-            "guard",
-            "if",
-            "import",
-            "in",
-            "init",
-            "inout",
-            "internal",
-            "is",
-            "let",
-            "nil",
-            "operator",
-            "optional",
-            "private",
-            "protocol",
-            "public",
-            "repeat",
-            "required",
-            "rethrows",
-            "return",
-            "self",
-            "static",
-            "struct",
-            "subscript",
-            "super",
-            "switch",
-            "throw",
-            "throws",
-            "true",
-            "try",
-            "typealias",
-            "var",
-            "where",
-            "while",
-]
+private let quotableEnumCases: Set<String> = {
+    () -> Set<String> in
+    var names: Set<String> = []
+
+    // We don't need to protect all of these keywords, just the ones
+    // that interfere with enum cases:
+    // names = names.union(PluginLibrary.swiftKeywordsReservedInParticularContexts)
+    names.insert("associativity")
+    names.insert("dynamicType")
+    names.insert("optional")
+    names.insert("required")
+
+    names = names.union(PluginLibrary.swiftKeywordsUsedInDeclarations)
```

Comment:
Maybe a comment why this only adds 3 of the sets instead of the 5 the others add?